weather.go

```diff
@@ -25,26 +27,23 @@ type Weather struct {
 		Pressure  int     `json:"pressure"`
 		Humidity  int     `json:"humidity"`
 	} `json:"main"`
+	Name string `json:"name"`
 }
 
 type CityUnknown struct {
 	Cod     string `json:"cod"`
 	Message string `json:"message"`
 }
 
-type CliResponse struct {
+type Conditions struct {
+	City    string
 	OneWord string  `json:"oneword"`
 	Celcius float64 `json:"celcius"`
 }
 
-type Response struct {
-	OneWord string  `json:"oneword"`
-	Celcius float64 `json:"celcius"`
-}
+func CliOutput(token string, location string) (output string) {
 
-func CliOutput(token, location string) (output string) {
-
-	var r Response
+	var r Conditions
 
 	resp := CallUrl(token, location)
```

Comment:
Since `resp` is a reader, don't we need to defer closing it here, or it'll leak?

Reply:
resp no longer reader